### PR TITLE
docs: Review and edit the docs files for the new docs website

### PR DESCRIPTION
**Pull request by anithamuthu2022**
The following changes are made to this PR: 
1. Reviewed all the files to make sure the `tier` info is correctly placed for each file. 
2. Added `<div class="enterprise-only">` for text that had ENT content. 
3. Changed the note message format to `!!! note`. 
4. Fixed the rendering issue on `ml_tutorials.md`
5. Fixed typos and added admonitions/notes in relevant pages/sections. 
6. Deleted `billing.md` page.

### CODE REVIEW

File: docs/source/guide/active_learning.md [hunk 1]
```diff
@@ -10,14 +10,14 @@ meta_description: Set up an end-to-end active learning loop with Label Studio us
 
 Follow this tutorial to set up an active learning loop with Label Studio. 
 
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class=enterprise-only>
```
Comment (huguestennier) [inline docs/source/guide/active_learning.md:13]:
```suggestion
<div class="enterprise-only">
```

File: docs/source/guide/install_enterprise_k8s.md [hunk 2]
```diff
@@ -12,9 +12,7 @@ Deploy Label Studio Enterprise on a Kubernetes Cluster using Helm 3. You can use
 
 Your Kubernetes cluster can be self-hosted or installed somewhere such as Amazon EKS. See the Amazon tutorial on how to [Deploy a Kubernetes Application with Amazon Elastic Container Service for Kubernetes](https://aws.amazon.com/getting-started/hands-on/deploy-kubernetes-app-amazon-eks/) for more about deploying an app on Amazon EKS.
 
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class="enterprise"><p>
```
Comment (huguestennier) [inline docs/source/guide/install_enterprise_k8s.md:15]:
```suggestion
<div class="enterprise-only"><p>
```

File: docs/source/guide/labeling.md [hunk 3]
```diff
@@ -1,10 +1,9 @@
 ---
 title: Label and annotate data 
 type: guide
-tier: all
 order: 402
 meta_title: Label and annotate data
-meta_description: "Label and annotate data to create bounding boxes, label text spans, set up relations. Filter and sort project data for machine learning dataset creation."
+meta_description: Label and annotate data using the Label Studio UI to create bounding boxes, label text spans, set up relations, and filter and sort project data for your machine learning dataset creation.
```
Comment (huguestennier) [inline docs/source/guide/labeling.md:6]: The original changes was a recommendation from Justin. Should we keep it?

File: docs/source/guide/labeling.md [hunk 4]
```diff
@@ -102,9 +101,7 @@ When annotators skip a task, the task no longer appears in the labeling queue fo
 
 In both Label Studio and Label Studio Enterprise, you can label tasks with collaborators. Tasks are locked while someone performs annotations so that you don't accidentally overwrite the annotations of another annotator. After the other annotator finishes with the task, it can appear in your queue for labeling if the minimum annotations per task is set to more than one. By default, tasks only need to be annotated by one annotator. 
 
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class="enterprise"><p>
```
Comment (huguestennier) [inline docs/source/guide/labeling.md:104]:
```suggestion
<div class="enterprise-only"><p>
```

File: docs/source/guide/manage_data.md [hunk 5]
```diff
@@ -55,9 +55,12 @@ You can create tabs on the data manager to [split your dataset](#Example-Split-a
 
 Tabs that you create depend on your [labeling configuration setup](setup.html), because the labeling configuration defines the data fields available for filtering and sorting. 
 
-> If you make changes to the labeling configuration, any tabs in your data manager are removed. Finish project setup before setting up complex tabs in the Data Manager.
+!!! note 
+    If you make changes to the labeling configuration, any tabs in your data manager are removed. Make sure to finish the project setup before setting up complex tabs in the Data Manager.
 
-## <i class='ent'></i> Assign annotators to tasks
+<div class="enterprise"><p>
```
Comment (huguestennier) [inline docs/source/guide/manage_data.md:61]:
```suggestion
<div class="enterprise-only"><p>
```

File: docs/source/guide/manage_users.md [hunk 6]
```diff
@@ -10,9 +10,7 @@ meta_description: Manage access and set up permissions with user roles, organiza
 
 Manage access to projects, organizations, and workspaces in Label Studio to restrict who can view data, annotations, and predictions in your data labeling projects. 
 
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class="enterprise"><p>
```
Comment (huguestennier) [inline docs/source/guide/manage_users.md:13]:
```suggestion
<div class="enterprise-only"><p>
```

File: docs/source/guide/predictions.md [hunk 7]
```diff
@@ -823,7 +823,7 @@ If annotators can't see predictions or if you encounter unexpected behavior afte
 In the **Settings > Machine Learning** section for your project, make sure that the following settings are configured:
 - Enable **Show predictions to annotators in the Label Stream and Quick View**
 - Select the relevant **Model Version** in the drop-down. If there is no drop-down menu visible, there might not be a model version listed for the pre-annotations, or there might be another issue happening. 
-- <i class='ent'></i> Disable the option to **Reveal pre-annotations interactively**, which requires manual action from annotators to display pre-annotated regions. (Label Studio Enterprise only)  
+- <div class="enterprise"><p> Disable the option to reveal pre-annotations interactively, which requires manual action from annotators to display pre-annotated regions.</p></div>
```
Comment (huguestennier) [inline docs/source/guide/predictions.md:826]:
```suggestion
- <div class="enterprise-only"><p> Disable the option to reveal pre-annotations interactively, which requires manual action from annotators to display pre-annotated regions.</p></div>
```

File: docs/source/guide/quality.md [hunk 8]
```diff
@@ -10,9 +10,7 @@ meta_description: In data labeling projects, start evaluating annotator performa
 
 After multiple labelers have annotated tasks, review their output to validate the quality of the results. You can also perform this task after a model has predicted labels for tasks in your dataset. To configure the settings for reviewing annotations, see [Set up review settings for your project](setup_project.html#Set-up-review-settings-for-your-project).
 
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class="enterprise"><p>
```
Comment (huguestennier) [inline docs/source/guide/quality.md:13]:
```suggestion
<div class="enterprise-only"><p>
```

File: docs/source/guide/security.md [hunk 9]
```diff
@@ -11,9 +11,7 @@ Label Studio provides many ways to secure access to your data and your deploymen
 
 All application component interactions are encrypted using the TLS protocol. 
 
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class="enterprise"><p>
```
Comment (huguestennier) [inline docs/source/guide/security.md:14]:
```suggestion
<div class="enterprise-only"><p>
```

File: docs/source/guide/setup_project.md [hunk 10]
```diff
@@ -36,9 +36,7 @@ After you save a project, any other collaborator with access to the Label Studio
 
 Set up annotation settings to configure how you want annotators to perform labeling for your project.
 
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class="enterprise"><p>
```
Comment (huguestennier) [inline docs/source/guide/setup_project.md:39]:
```suggestion
<div class="enterprise-only"><p>
```

**Review by huguestennier — CHANGES_REQUESTED** [review]
I think all `<div class="enterprise">` should be changed to `<div class="enterprise-only">` right?

File: docs/source/guide/labeling.md (hunk 3 above)
Comment (anithamuthu2022) [reply docs/source/guide/labeling.md:6]: yes

**Comment by anithamuthu2022** [comment]
@huguestennier could you help fix the merge conflict on docs/themes/v2/layout/partials/header.ejs?

**Comment by anithamuthu2022** [comment]
@huguestennier could you please review this commit to make sure everything looks good/requires a change? 
https://github.com/heartexlabs/label-studio/pull/3253/commits/f3785ddd759017302680b82b9d974c39b7be16ed

File: docs/source/guide/styleguide-enterprise.md [hunk 11]
```diff
@@ -0,0 +1,430 @@
+---
+title: Styleguide for Enterprise
+short: Styleguide-Enterprise
+tier: enterprise
+type: guide
```
Comment (makseq) [inline docs/source/guide/styleguide-enterprise.md:5]: let's remove this line, this md file should be hidden in our docs

Comment (makseq) [reply docs/source/guide/styleguide-enterprise.md:5]: the same for `styleguide.md`

**Review by makseq — CHANGES_REQUESTED** [review]

**Comment by anithamuthu2022** [comment]
I've removed `type: guide` (line #4) from the following files so the testing pages are not visible on the left nav
another-subpage.md
test-sub-page.md

**Comment by anithamuthu2022** [comment]
This page `ts+video.md` is missing from the docs site. This also looks like a standalone page like the `pingone.md` file. 
https://github.com/heartexlabs/label-studio/blame/develop/docs/source/guide/ts%2Bvideo.md
https://github.com/heartexlabs/label-studio/blame/develop/docs/source/guide/pingone.md

Let me know if the `order` for these pages to get included in the docs website.

**Comment by anithamuthu2022** [comment]
`decommission_minio.md` is also a standalone page and is not visible on the docs website.

**Comment by anithamuthu2022** [comment]
@huguestennier @makseq I've addressed your feedback and also replaced the format for `Note` messages. 
When you get a chance please review this PR and we can merge it. Thank you!

**Comment by makseq** [comment]
> This page `ts+video.md` is missing from the docs site. This also looks like a standalone page like the `pingone.md` file. https://github.com/heartexlabs/label-studio/blame/develop/docs/source/guide/ts%2Bvideo.md https://github.com/heartexlabs/label-studio/blame/develop/docs/source/guide/pingone.md
> 
> Let me know the `order` for these pages to get included in the docs website.
> 
> The file naming format for `ts+video.md` is incorrect. I've renamed it as `timeseries_video.md`.

yes, it should be accessible on demand only

**Review by makseq — APPROVED** [review]

**Pull request merged**